log4perl_prod.conf

```diff
@@ -4,14 +4,30 @@ log4perl.filter.GazelleWarnings                 = Log::Log4perl::Filter::StringM
 log4perl.filter.GazelleWarnings.StringToMatch   = failed to writev: -1 errno:32
 log4perl.filter.GazelleWarnings.AcceptOnMatch   = false
 
-log4perl.appender.OUTPUT                            = Log::Log4perl::Appender::Screen
-log4perl.appender.OUTPUT.stderr                     = 1
-log4perl.appender.OUTPUT.Filter                     = GazelleWarnings
-log4perl.appender.OUTPUT.layout                     = PatternLayout
-log4perl.appender.OUTPUT.layout.ConversionPattern   = [%d] [%p] [%X{url}] %m%n
+log4perl.filter.NetworkErrors                   = Log::Log4perl::Filter::StringMatch
+log4perl.filter.NetworkErrors.StringToMatch     = Timed out at
```

Comment:
Do we really want to filter out messages like this? If we're getting timeouts, we should be seeing those errors.